chain/ethereum/src/data_source.rs

```diff
@@ -217,6 +225,8 @@ impl blockchain::DataSource<Chain> for DataSource {
             manifest_idx,
             address,
             start_block: 0,
+            // EBTODO: Re-evaluate if this needs to be set to done_at,
+            end_block: done_at,
```

Comment:
As far as I understand, done_at is only used for file data sources, it does not make sense to use it here as there is no "done" for a Dynamic Data Source in other scenarios. Would be good to confirm with @leoyvens as he may have intended this to be used in some other and add a comment to explain what it is and what it is used for to avoid confusion in the future.

Reply:
This is to say I don't think we should use this field as that will make it quite difficult to understand the meaning on different contexts, I'd rather just add an explicit field end_block or maybe consider deleting the data source once it's no longer needed?

Reply:
I just set the field to `None` for now, but if we decide to have a way to have an `endBlock` for dynamic data sources, we would need to use `done_at` at some point and then set `endBlock` here.